src/main/java/tasklist/DeleteTask.java

```diff
@@ -0,0 +1,47 @@
+package tasklist;
+
+import java.util.List;
```

Comment:
Like that you split the taskList into individual classes for each command! 💯

Reply:
Like that you have specific classes to handle different commands! 💯